usaspending_api/awards/management/commands/compare_table_to_materialized_view.py

```diff
@@ -0,0 +1,208 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from functools import partial
+from psycopg2.sql import Identifier, Literal, SQL
+from usaspending_api.common.helpers.sql_helpers import convert_composable_query_to_string, get_connection
+from usaspending_api.common.helpers.timing_helpers import Timer
+
+
+logger = logging.getLogger("console")
+
+
+# All of our timers log to logger.info and logger.error.
+Timer = partial(Timer, success_logger=logger.info, failure_logger=logger.error)
+
+
+DIFF_LIMIT = 50
+
+
+class Command(BaseCommand):
+
+    schema_name = "public"
+
+    help = (
+        "Compare materialized view to table. A little tool we're using to monitor "
+        "our materialized view to table conversion. Uses row hash to compare, so the "
```

Comment:
Note: If we change the approach to add additional columns for joined-table-primary-key identifiers ... the columns between matview and tableview will be different... rendering different row hashes. 

Maybe we can assume the tableview is a superset of the matview, so use the matview cols to index which columns of the tableview get appended to a string for the md5 hash?

Reply:
My plan was to add a `--exclude` switch to allow certain columns to be ignored.  We can discuss further if/when we continue this work.